problems/basic_test.go

```diff
@@ -0,0 +1,109 @@
+package problems
+
+import (
+	"context"
+	"crypto/rand"
+	"net/http"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/sanity-io/litter"
+	"go.opencensus.io/trace"
+)
+
+func TestBasicProblem_DecorateWithRequest(t *testing.T) {
+	rand.Reader = strings.NewReader("abcdefghijklmnopqrstuvwxyz")
```

Comment:
Do we need this?

Reply:
If we don't do this we get a new random trace id each time

Reply:
We could remove it and then assert on each field and just assert that it's not empty string instead?